src/redux/actions/ForgotPasswordAction.js

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import * as ActionTypes from "../action_types";
+import API_BASE_URL from "./index";
+
+/**
+ * request to change password
+ *
+ * @param {string} email
+ * @return {object} type and responseData
+ */
+export const forgotPassword = responseData => ({
+  type: ActionTypes.FORGOT_PWD,
+  responseData
+});
+
+/**
+ * error from forgot password request
+ *
+ * @param {string} error
+ * @return {object} type and responseData,{boolean} info
+ */
+
+export const forgotPasswordError = error => ({
+  type: ActionTypes.ERROR,
+  error
+});
+
+/**
+ * send request to generate reset password link
+ *
+ * @param {object} email
+ * @return {function} dispatch
+ */
+
```

Comment:
Remove this extra line after the comment. Line 34, line 22 and the rest. Attention to detail.